static/ankersrv.js

```diff
@@ -40,15 +66,46 @@ $(function () {
         }
     });
 
-    document.addEventListener("fullscreenchange", function() {
-        /* Make more room for video element in fullscreen mode */
-        if (document.fullscreenElement) {
+    /**
+     * Sets or unsets the required classes for fullscreen functionality.
+     *
+     * @function
+     * @param {boolean} fullscreen - Whether to set or unset the classes (true to set, false to unset).
+     * @param {boolean} emulate - Whether to emulate the fullscreen mode or not.
+     */
+    function setFullscreenClasses(fullscreen = false, emulate = false) {
+        if (fullscreen) {
+            if (emulate) {
+                $("#vmain").addClass("fullscreen-emulate");
+                $("body").addClass("fullscreen-emulate");
+                $("header").addClass("fullscreen-emulate");
+                $("footer").addClass("fullscreen-emulate");
```

Comment:
We can simplify this, by using css classes a bit more.

Let's put the `fullscreen-emulate` on these 4 things we are selecting here.

Then, we can make a css class `fullscreen-emulate-active`. The js then becomes

```js
if (emulate)
    $(".fullscreen-emulate").addClass("fullscreen-emulate-active");
```

(and vice versa for teardown)

That way, we can tag any dom element we want to apply special rules for, without changing the js in the future.

Reply:
ahh, that would be easier :D

Reply:
✅